src/test/java/gov/usgs/wma/waterdata/collections/CollectionParamsTest.java

```diff
@@ -0,0 +1,65 @@
+package gov.usgs.wma.waterdata.collections;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static gov.usgs.wma.waterdata.collections.CollectionParams.*;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import gov.usgs.wma.waterdata.ConfigurationService;
+
+class CollectionParamsTest {
+
+	private static final String TEST_SERVER_URL="/sever/url";
+
+	private String expectedCollectionId = "aCollectionId";
+	private String expectedFeatureId = "aFeatureId";
+	private String expectedTimeSeriesId = "aGUID";
+
+	private ConfigurationService config;
+	private CollectionParams builder;
+	private Map<String,Object> params;
```

Comment:
Typically the spring components are autowired in:
@Autowired
private ConfigurationService config;

Reply:
Ah, yes. I am open to that. That would be an integration test. Autowired is not available in my unit test, no spring context started. I was able to get away with a unit test rather than a integration test. The reason I mocked it this way was that they were easily mocked on their own. One was just a set of values and the other just only depended on that source of values. I have isolation this way. Now we could do the Autowired in an Integration test.

Reply:
It has been agreed that this will be done when there is an integration test.